### DISCUSSION TITLE
视频播放完成后自动停止

### DISCUSSION THREAD

**Discussion by Tyrant1984jr** [the1812/Bilibili-Evolved#1318] [综合讨论 / General]
现在B站网页视频播放完成后会自动倒计时进入下一个视频播放，需要手动取消。希望可以增加一个永久取消自动播放下一条的功能。谢谢。

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/102201026-8b9ffd00-3f00-11eb-9ad8-25193810f6ea.png)

**Reply by Tyrant1984jr** [reply]
谢谢，我去看看一下这个设置。

**Comment by Coulomb-G** [comment]
你要的开关,b站就有,在最右侧的推荐视频栏,如果屏蔽了推荐视频,可以先取消屏蔽来关闭该功能

**Reply by Tyrant1984jr** [reply]
谢谢，我去找一下。

**Comment by LonelySteve** [comment]
@Tyrant1984jr 参见 #1285，对于那些以前就隐藏了推荐视频的人来说，这个问题真的会一头雾水

**Reply by Tyrant1984jr** [reply]
确实是，我头疼了好久……每次都要眼疾手快去点那个“取消”。
